recipes/zstd/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "zstd" %}
+{% set version = "1.1.3" %}
+{% set sha256hash = "106c532ae840a6ee4aee5258f04f3acab7b3e09b9e9584ebe94e4fbfd899af0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/facebook/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256hash }}
+  patches:
+    - patches/0001-Fix-VS2008-solution-format-version.patch  # [win]
+    - patches/0002-Fix-order-of-building-for-VS2010.patch    # [win]
```

Comment:
Probably worthwhile to send these upstream. Doesn't need to be completed right away, but it would be good to get straightened out.